composer.json

```diff
@@ -43,7 +43,7 @@
         "phpunit/php-code-coverage": "^10.1.1",
         "phpunit/php-file-iterator": "^4.0.1",
         "phpunit/php-timer": "^6.0",
-        "phpunit/phpunit": "^10.1.2",
+        "phpunit/phpunit": ">= 10.1.2 <10.2",
```

Comment:
I've added expicit support for PHPUnit 10.2 in https://github.com/paratestphp/paratest/pull/767: can you rebase a remove this line please?

Reply:
Hi @Slamdunk  , rebased and applied the changes according to your request.
Let me know if it is ok from your side.

Reply:
Hi @Slamdunk , rebased and applied the changes. Tests from php 8.1 to 8.3 are green ✅

Reply:
Mmm, something went wrong.

I see you didn't create a separate branch for this changes, so I don't know what happened when you say "_rebased and applied the changes_".

If the rebase had went as expected, we wouldn't see the changes in `src/WrapperRunner/ApplicationForWrapperWorker.php` perhaps.

Can you try moving the edits in a separate branch, rebase it onto the `upstream/7.x` e push it here please?

Reply:
Rebasing the main branch, i missed the "Handle unexpected output (#769)".
Now as you can see, the small differences (with php8.3) between my branch and the upstream are:

https://github.com/paratestphp/paratest/pull/766/files